public/app/features/connections/pages/EditDataSourcePage.tsx

```diff
@@ -4,18 +4,41 @@ import { useLocation, useParams } from 'react-router-dom';
 import { Page } from 'app/core/components/Page/Page';
 import { EditDataSource } from 'app/features/datasources/components/EditDataSource';
 import { EditDataSourceActions } from 'app/features/datasources/components/EditDataSourceActions';
-
-import { useDataSourceSettingsNav } from '../hooks/useDataSourceSettingsNav';
+import { EditDataSourceTitle } from 'app/features/datasources/components/EditDataSourceTitle';
+import { useDataSourceInfo } from 'app/features/datasources/components/useDataSourceInfo';
+import { useDataSourceSettingsNav } from 'app/features/datasources/state';
 
 export function EditDataSourcePage() {
   const { uid } = useParams<{ uid: string }>();
   const location = useLocation();
   const params = new URLSearchParams(location.search);
   const pageId = params.get('page');
-  const { navId, pageNav } = useDataSourceSettingsNav();
+  const nav = useDataSourceSettingsNav(uid, pageId);
+
+  const info = useDataSourceInfo({
+    dataSource: nav.dataSource,
+    dataSourcePluginName: nav.main.dataSourcePluginName,
+    isDefault: nav.dataSource.isDefault,
+    alertingSupported: nav.dataSourceHeader.alertingSupported,
+    onUpdate: nav.dataSourceHeader.onUpdate,
+  });
 
   return (
-    <Page navId={navId} pageNav={pageNav} actions={<EditDataSourceActions uid={uid} />}>
+    <Page
+      navId="connections-your-connections-datasources"
+      pageNav={nav.main}
+      renderTitle={(title) => (
+        <EditDataSourceTitle
+          dataSource={nav.dataSource}
+          title={title}
+          readOnly={nav.dataSource.readOnly}
+          onUpdate={nav.dataSourceHeader.onUpdate}
+        />
+      )}
+      subTitle={<></>}
```

Comment:
why this? Is it to remove the description line that has "Type: Prometheus", can't we remove that where the nav model is built instead?